src/fhirtypes/common.ts

```diff
@@ -19,6 +20,14 @@ export function setPropertyOnInstance(
     value,
     resolve
   );
+  setFixedValueOfPath(instance, pathParts, fixedValue);
+}
+
+export function setFixedValueOfPath(
```

Comment:
Could you just add a comment header to this function? Also I know I stole the name that this function should have, but maybe we can come up with better names for this function and `setPropertyOnInstance`. I feel like `setFixedValueOfPath` should truly be named `setPropertyOnInstance`. Maybe then `setPropertyOnInstance` could become `setPropertyOnDefinitionInstance`? I guess still kinda confusing but maybe a little more clear.

Reply:
I agree the naming is confusing. Maybe the existing `setPropertyOnInstance` could be renamed to `setPropertyOnStructureDefinitionInstance`? @ngfreiter do you think that's too wordy? Then the `setFixedValueOfPath` could actually be `setPropertyOnInstance`.

Reply:
The existing `setPropertyOnInstance` gets used on both `StructureDefinition`s and `ElementDefinition`s. So the most accurate name would probably be `setPropertyOnStructureDefinitionOrElementDefinitionInstance`. But now that is pretty dang wordy, which is why the best I could come up with was `setPropertyOnDefinitionInstance`, but that isn't as clear.

Reply:
Hm I honestly don't mind `setPropertyOnDefinitionInstance` and I definitely prefer it over my suggestion now that you pointed that out. So I'm good with using that, as long as you don't find it too unclear.